tls_codec/src/quic_vec.rs

```diff
@@ -251,18 +254,6 @@ macro_rules! impl_vl_bytes_generic {
             pub fn as_slice(&self) -> &[u8] {
                 self.vec().as_ref()
             }
-
-            /// Add an element to this.
-            #[inline]
-            pub fn push(&mut self, value: u8) {
```

Comment:
These are public functions. We should keep them. Deleting them would be a breaking change. Let's just ignore them for hax extraction for now.